core/domain/question_domain.py

```diff
@@ -244,16 +222,32 @@ def update_question_data(self, question_data):
 class QuestionSummary(object):
     """Domain object for Question Summary.
     """
-    def __init__(self, question_id, question_content):
+    def __init__(
+            self, question_id, creator_id, language_code, status,
+            question_html_data, question_model_last_updated=None,
+            question_model_created_on=None):
         """Constructs a Question Summary domain object.
 
         Args:
             question_id: str. The ID of the question.
-            question_content: str. The static HTML of the question shown to
+            creator_id: str. The user ID of the creator of the question.
+            language_code: str. The code that represents the question
```

Comment:
Modify this to be similar to other language code docstrings, specifying the type of code.

Reply:
done